content/chainguard/chainguard-images/getting-started/nginx/index.md

```diff
@@ -0,0 +1,262 @@
+
+```shell
+mkdir data && cd $_
+```
+
+Using a text editor of your choice, create a new file `index.html` for the HTML content that will be served. We will use `nano` as an example.
+
+```shell
+nano index.html
+```
+
+The following HTML file displays an image of Inky alongside a fun octopus fact.
+
+```HTML
+<!DOCTYPE html>
+<html>
+<head>
+<title>Chainguard nginx Demo Application</title>
+<link rel="stylesheet" href="stylesheet.css">
+</head>
+
+<body>
+
+<h1>nginx Demo Application</h1>
+
+<h2>from the <a href="https://edu.chainguard.dev/" target="_blank">Chainguard Academy</a></h2>
+
+<img src="inky.png" class="corners" width="250px">
+
+<i><h3>Did you know?</h3></i>
+<p>The Wolfi octopus is the world's smallest octopus, weighing in on average at less than a gram!</p>
+<p>They are found near the coastlines of the west Pacific Ocean.</p>
+
+</body>
+
+</html>
+```
+
+Copy this code to your `index.html` file, save, and close it.
+
+Next, create a CSS file named `stylesheet.css` using the text editor of your choice. We'll use `nano` to demonstrate.
+
+```shell
+nano stylesheet.css
+```
+
+Copy the following code to your `stylesheet.css` file. 
+
+```CSS
+/*
+Chainguard Academy nginx Demo Application
+*/
+
+body {
+    text-align: center;
+    background-color: #fcebfc;
+    font-family: Arial, sans-serif;
+}
+
+h1 {
+    color: #df45e6;
+}
+
+h2, h3 {
+    color: #9745e6;
+}
+
+p {
+    color: #583ce8
+}
+
+.corners {
+    border-radius: 10px;
+}
+
+```
+
+After copying the code into the `stylesheet.css` file, save and close it.
+
+Next, you will pull down the `inky.png` file using `curl`. Always [inspect the URL](https://raw.githubusercontent.com/chainguard-dev/edu-images-demos/734e6171ee69f0e0dbac95e6ebc1759ac18bf00a/nginx/data/inky.png) before downloading it to ensure it comes from a safe and trustworthy location.
+
+```shell
+curl -O https://raw.githubusercontent.com/chainguard-dev/edu-images-demos/734e6171ee69f0e0dbac95e6ebc1759ac18bf00a/nginx/data/inky.png
+```
+
+Now, return to the `nginxdemo` directory you made earlier. Here we will create the `nginx.conf` configuration file used by nginx to run the local web server. We will demonstrate this using `nano`.
+
+```shell
+nano nginx.conf
+```
+
+Copy the following code into the configuration file you created. In the `location` directive, be sure to update the configuration to reference the path from root to the `data` directory you created in a previous step.
+
+```nginx
+worker_processes  1;
+
+events {
+    worker_connections  1024;
+}
+
+http {
+    include       mime.types;
+    default_type  application/octet-stream;
+
+    sendfile        on;
+
+    keepalive_timeout  65;
+
+    server {
+
+        listen 1313;
+        server_name localhost;
+        charset koi8-r;
+
+        location / {
+            root /Users/username/nginxdemo/data; # Update the file path for your system
```

Comment:
There's kind of no way around these absolute paths in this tutorial, but you may want to put a note in somewhere with an example of how it would look on Linux, i.e. "/home/username/"